functions/src/custom/event.ts

```diff
@@ -53,7 +52,7 @@ export const create_event = async (document: FirebaseFirestore.DocumentData): Pr
         first_name: first_name,
         last_name: last_name,
         name: name,
-        checkin_link: `https://${environment.URL_PROD}/checkin/${path_name}`,
+        checkin_link: `${environment.URL_PROD}/checkin/${path_name}`,
         date: date,
```

Comment:
Has the checkin-link begun including "https://" by default then?

Reply:
`URL_PROD` is stored as `https://portal.acmutd.co` in Doppler. Avoids having to have a function to add `https://` for each element in the cors array. The things that don't conform to having `https://` in Doppler are `AUTH0_DOMAIN` and `CLOUDFLARE_DOMAIN` but thats to let us copypasta straight from what they provide within their respective consoles.